1d6-osr-ultra-lite-and-small-dungeon.md

```diff
@@ -45,4 +45,4 @@ Alternativelly, print this image and cut through the black lines. Shuffle the 12
 
 ----
 
-Source: http://daylands.blogspot.fr/2017/11/1d6-osr-ultra-lite-small-dungeon.html
+Source: <http://daylands.blogspot.fr/2017/11/1d6-osr-ultra-lite-small-dungeon.html>
```

Comment:
can't see why the angle brackets are required here...

Reply:
Oh, I see, on the Github pages, it's not transformed into a link... thx for this fix.